includes/class-wc-embed.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Embed Class which handles any WooCommerce Products that are embedded on this site or another site
+ *
+ * @class 		WC_Embed
+ * @version		2.5.0
+ * @package		WooCommerce/Classes/Embed
+ * @category	Class
+ * @author 		WooThemes
+ */
+class WC_Embed {
+
+	/** @var WC_Embed The single instance of the class */
+	protected static $_instance = null;
+
+	/**
+	 * Main WC_Embed Instance.
+	 *
+	 * Ensures only one instance of WC_Embed is loaded or can be loaded.
+	 *
+	 * @since 2.5
+	 * @static
+	 * @return WC_Embed Main instance
+	 */
+	public static function instance() {
```

Comment:
You don't need this method, since you init the class using `WC_Embed::init()` :)